src/Nest/ElasticClient-Scroll.cs

```diff
@@ -56,7 +58,9 @@ public partial class ElasticClient
 				{
 					string scrollId = p.ScrollId;
 					p.ScrollId = null;
-					return this.RawDispatch.ScrollDispatchAsync<SearchResponse<T>>(p, scrollId);
+				    p.DeserializationState(CreateScrollDeserializer<T, T>(d));
+
+                    return this.RawDispatch.ScrollDispatchAsync<SearchResponse<T>>(p, scrollId);

```

Comment:
This should go on the other overloads as well. Can you also make sure your PR uses tabs not spaces?